_config/tinymce.yml

```diff
@@ -2,6 +2,7 @@
 Name: admintinymce
 ---
 SilverStripe\Forms\HTMLEditor\TinyMCEConfig:
-  base_dir: 'silverstripe/admin:thirdparty/tinymce'
+  base_dir: 'silverstripe/admin:client/dist/tinymce'
+  lang_dir: 'silverstripe/admin:thirdparty/tinymce_lang'
```

Comment:
tinymce itself is in `client/dist/` to reflect the fact that it's being put there as part of `yarn build` - i.e. it's not just pasted in there directly, which means we'll get updates through the normal dependency update mechanisms.

The lang dir is in `thidrparty/` to reflect the fact that it's third party code which we've downloaded and dumped in there - it's not grabbed as part of `yarn build` so to updates to this will be manual.